File: app/src/main/java/com/elephantgroup/one/ui/home/MainContactFragment.java
package com.elephantgroup.one.ui.home;

import com.elephantgroup.one.R;
import com.elephantgroup.one.base.BaseFragment;

public class MainContactFragment extends BaseFragment<MainContactContract.Presenter> implements MainContactContract.View  {

    @Override
    public int getLayoutId() {
        return R.layout.main_contact_layout;
    }

    @Override
    public MainContactPresenter createPresenter() {
        return new MainContactPresenter(this);
    }

    @Override
    protected void initData() {
        setTitle(getString(R.string.main_contact));
    }

    @Override
    public void onResult(Object result, String message) {

    }

    @Override
    public void onError(Throwable throwable, String message) {

    }
}

File: app/src/main/java/com/elephantgroup/one/ui/home/MainContactFragment.kt
package com.elephantgroup.one.ui.home

import com.elephantgroup.one.R
import com.elephantgroup.one.base.BaseFragment

class MainContactFragment : BaseFragment<MainContactContract.Presenter>(), MainContactContract.View {

    override fun getLayoutId(): Int {
        return R.layout.main_contact_layout
    }

    override fun createPresenter(): MainContactPresenter {
        return MainContactPresenter(this)
    }

    override fun initData() {
        setTitle(getString(R.string.main_contact))
    }

    override fun onResult(result: Any, message: String) {

    }

    override fun onError(throwable: Throwable, message: String) {

    }
}
